caikit_nlp/modules/text_generation/peft_tgis_remote.py

```diff
@@ -320,6 +341,28 @@ def run_tokenizer(
             TokenizationResults
                 The token count
         """
+        if self._tgis_backend:
+            self._register_model_connection_with_context(context)
+
         return self.tgis_generation_client.unary_tokenize(
             text=text,
         )
+
+    def _register_model_connection_with_context(
+        self, context: Optional[RuntimeServerContextType]
+    ):
+        """
+        Register a model connection with the configured TGISBackend.
+        """
+        ok, route_info = get_route_info(context)
+        if ok:
```

Comment:
I really want to be able to use `:=` here and make it look more like `go`, but seems that [you can't](https://stackoverflow.com/a/59568122)